src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/LocalsTests.cs

```diff
@@ -3842,6 +3842,43 @@ static void DummySequencePoint()
             });
         }
 
+        [WorkItem(55606, "[https://github.com/dotnet/roslyn/issues/55606](https://github.com/dotnet/roslyn/issues/55606)")]
```

Comment:
Sorry, the link format in my comment was incorrect. The attribute should be:
`[WorkItem(55606, "https://github.com/dotnet/roslyn/issues/55606")]`